apps/base-docs/docusaurus.config.js

```diff
@@ -185,6 +185,8 @@ const config = {
           to: '/tutorials',
           navposition: 'bottomLeft',
           label: 'Tutorials',
+          type: 'custom-navbarLink',
+          eventLabel: 'TutorialsButton_Clicked',
```

Comment:
Did we want to try to align event naming with the best practices as mentioned in documentation? https://docs.cbhq.net/frontend/analytics/log-events#best-practices

Reply:
Yes, Gardo is planning to give me updated event names, hopefully today.